src/components/Cluster/ClusterDetail/NodePool.tsx

```diff
@@ -35,12 +38,23 @@ const NPViewAndEditNameStyled = styled<
   }
 `;
 
+interface INodesWrapperProps {
+  highlight?: boolean;
+  children: React.ReactNode;
+}
+
 const NodesWrapper = styled.div`
```

Comment:
I think I know what you meant
```suggestion
const NodesWrapper = styled<{highlight?: boolean}>.div`
```

This should do it

Reply:
thanks :pray: